DatadogInternal/Sources/Models/RUM/RUMCoreContext.swift

```diff
@@ -34,12 +36,22 @@ public struct RUMCoreContext: AdditionalContext, Equatable {
         sessionID: String,
         viewID: String? = nil,
         userActionID: String? = nil,
-        viewServerTimeOffset: TimeInterval? = nil
+        viewServerTimeOffset: TimeInterval? = nil,
+        webViewServerTimeOffsets: [String: TimeInterval] = [:]
     ) {
         self.applicationID = applicationID
         self.sessionID = sessionID
         self.viewID = viewID
         self.userActionID = userActionID
         self.viewServerTimeOffset = viewServerTimeOffset
+        self.webViewServerTimeOffsets = webViewServerTimeOffsets
+    }
+
+    public func serverTimeOffset(forWebView id: String) -> TimeInterval? {
```

Comment:
suggestion/ The offset is associated to a RUM view, not to the webview :) I would rename this to 
```suggestion
    public func serverTimeOffset(forView id: String) -> TimeInterval? {
```